python/processors/nft_marketplace_v2/constants.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+MARKETPLACE_V2_ADDRESS = (
+    "0xb11affd5c514bb969e988710ef57813d9556cc1e3fe6dc9aa6a82b56aee53d98"
+)
```

Comment:
this should be in a config.

Reply:
What's wrong with hardcoding this? The other token v1 marketplace processor parses a ton of marketplaces and if we had those addresses in config, that'd explode the # of config vars.